functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,14 @@
  */
 struct BeerOrganizer
 {
+    BeerBrand beer = BeerBrand::None;
+
+    BeerBrand operator()()
+    {
+        beer = static_cast<BeerBrand>(static_cast<int>(beer) + 1) ;
+        if(beer == BeerBrand::Max)beer = BeerBrand::HoeGaarden;
```

Comment:
Please follow this style, it's much readable
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es85-make-empty-statements-visible